src/TextInput/index.js

```diff
@@ -33,6 +33,7 @@ class TextInput extends Control {
                         type={this.props.type}
                         disabled={this.props.disabled}
                         placeholder={this.props.placeholder}
+                        autoComplete="off"
```

Comment:
Let's add an ability to `off` value from the props. It doesn't feel wise to disallow it completely.